package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
```

Comment:
We shouldn't need this line since we are using styled-components library for styling with MaterialUI instead of emotion